packages/Search/src/details/DTK_DetailsKokkosExt.hpp

```diff
@@ -13,8 +13,18 @@
 
 #include <Kokkos_View.hpp>
 
+#include <cmath>   // isfinite, HUGE_VAL
+#include <cstdint> // uint32_t
 #include <type_traits>
 
+#if __cplusplus < 201402L
+namespace std
```

Comment:
is it safe to have our own `namespace std` code? does this do something that is already in C++14 so it is interchangeable when we use C++11?

Reply:
> is it safe to have our own `namespace std` code? does this do something that is already in C++14 so it is interchangeable when we use C++11?

It is undefined behavior to add declarations to the namespace `std` which means we don't have the guarantee the standard library won't break if we do such thing.  This is just a cheap way to use this feature from C++14.  It is not pretty but it does what I want.